ios/PurchasesHybridCommon/PurchasesHybridCommon.xcodeproj/project.pbxproj

```diff
@@ -1299,6 +1307,25 @@
 			defaultConfigurationName = Release;
 		};
 /* End XCConfigurationList section */
+
+/* Begin XCRemoteSwiftPackageReference section */
+		57D6691C28DD19D900622DE6 /* XCRemoteSwiftPackageReference "swift-snapshot-testing" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/pointfreeco/swift-snapshot-testing";
+			requirement = {
+				kind = upToNextMajorVersion;
+				minimumVersion = 1.9.0;
```

Comment:
I guess we don't have tests with < iOS 13 in PHC right?

Reply:
Good point, yeah we don't. I think since we do test the underlying SDK in iOS 12+ it's fine if the PHC is only tested in one version.